packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -397,6 +398,9 @@ class DDSCarousel extends HostListenerMixin(StableSelectorMixin(LitElement)) {
   @property({ attribute: false })
   formatStatus = ({ currentPage, pages }) => `${currentPage} / ${pages}`;
 
+  @property({ reflect: true, attribute: 'in-modal' })
+  inModal?: String;
```

Comment:
Wondering if this might be better as a Boolean attribute?

Reply:
We never actually check this property, it's used solely for styling. I _think_ i went with an empty string to keep the HTML concise (`[in-modal]` vs `[in-modal="true"]`)